libtcmu.c

```diff
@@ -347,6 +821,20 @@ static void remove_device(struct tcmulib_context *ctx,
 
 	darray_remove(ctx->devices, i);
 
+	struct tcmu_io_queue *io_wq = &dev->work_queue;
```

Comment:
Could you move this with the other variable declarations so we have similar coding style.